app/assets/stylesheets/common/admin/dashboard.scss

```diff
@@ -246,16 +246,21 @@
     .problem-messages {
       margin-bottom: 1em;
 
-      &.priority-high {
-        background-color: var(--danger-low);
-        border: 1px solid var(--danger-medium);
-      }
```

Comment:
Tone down the styling of high priority, because it was pretty crazy, and the button renders extremely poorly on the background. It still has the red warning triangle icon.